pkg/virt/types.go

```diff
@@ -0,0 +1,14 @@
+package virt
```

Comment:
Could you please add the file header with its reference to LICENSE information? I like the idea of an interface!

Reply:
Sure, will do later